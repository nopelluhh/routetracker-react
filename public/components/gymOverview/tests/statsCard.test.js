import StatsCard from '../components/statsCard'

import React from 'react'
import renderer from 'react-test-renderer'

describe('statsCard tests', () => {
    const testData = [
        {setOn: new Date()}
    ]

    it('should adjust style based on props', () => {

        const component = renderer.create(
            <StatsCard data={testData}> Facebook </StatsCard>
        )

        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
