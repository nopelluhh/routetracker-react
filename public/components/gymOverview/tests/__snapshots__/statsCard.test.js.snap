// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`statsCard tests should adjust style based on props 1`] = `
<div
  className="rt-pane "
>
  <span
    className="rt-pane__header"
  >
    Overview
  </span>
  <table
    className="table"
  >
    <tbody>
      <tr>
        <td
          className="text-right"
        >
          Count:
        </td>
        <td
          className="text-left"
        >
          1
        </td>
      </tr>
      <tr>
        <td
          className="text-right"
        >
          Routes Needed:
        </td>
        <td
          className="text-left"
        >
          N/A
        </td>
      </tr>
      <tr>
        <td
          className="text-right"
        >
          Oldest Route:
        </td>
        <td
          className="text-left"
        >
          Invalid Date
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
